pkg/services/authz/zanzana/server/server.go

```diff
@@ -26,11 +26,16 @@ var _ authzextv1.AuthzExtentionServiceServer = (*Server)(nil)
 
 var tracer = otel.Tracer("github.com/grafana/grafana/pkg/services/authz/zanzana/server")
 
+type OpenFGAServer interface {
+	openfgav1.OpenFGAServiceServer
+	IsReady(ctx context.Context) (bool, error)
```

Comment:
Where is it implemented? Cannot find.

Reply:
https://github.com/openfga/openfga/blob/main/pkg/server/server.go#L828